src/sentry/static/sentry/app/chartcuterieConfig.tsx

```diff
@@ -0,0 +1,103 @@
+/* global process */
+
+/**
+ * This module is used to define the look and feels for charts rendered via the
+ * backend chart rendering service Chartcuterie.
+ *
+ * Be careful what you import into this file, as it will end up being bundled
+ * into the configuration file loaded by the service.
+ */
+
+import type {
+  ChartcuterieConfig,
+  RenderConfig,
+  RenderDescriptor,
+} from '@sentry/chartcuterie/lib/types';
+
+import Grid from 'app/components/charts/components/grid';
+import Legend from 'app/components/charts/components/legend';
+import XAxis from 'app/components/charts/components/xAxis';
+import YAxis from 'app/components/charts/components/yAxis';
+import AreaSeries from 'app/components/charts/series/areaSeries';
+import {getColorPalette} from 'app/components/charts/utils';
+import {EventsStatsData} from 'app/types';
+import {lightTheme as theme} from 'app/utils/theme';
+
+/**
+ * Defines the keys which may be passed into the chartcuterie chart rendering
+ * service.
+ */
+export enum ChartType {
+  SLACK_DISCOVER_TOTAL_PERIOD = 'slack:discover.totalPeriod',
+}
+
+/**
+ * All registered style descriptors
+ */
+const renderConfig: RenderConfig<ChartType> = new Map();
+
+/**
+ * Register a style descriptor
+ */
+const register = (renderDescriptor: RenderDescriptor<ChartType>) =>
+  renderConfig.set(renderDescriptor.key, renderDescriptor);
+
+/**
+ * Slack unfurls for discover using the Total Period view
+ */
+register({
+  key: ChartType.SLACK_DISCOVER_TOTAL_PERIOD,
+  height: 150,
+  width: 450,
+  getOption: (data: EventsStatsData) => {
+    const color = getColorPalette(theme, data.length);
+
+    const series = {
+      seriesName: 'Current',
+      data: data.map(([timestamp, countsForTimestamp]) => ({
+        name: timestamp * 1000,
+        value: countsForTimestamp.reduce((acc, {count}) => acc + count, 0),
+      })),
+    };
+
+    return {
+      useUTC: true,
+      color,
+      backgroundColor: '#fff',
+      grid: Grid({left: 5, right: 5, bottom: 5}),
+      legend: Legend({theme, itemHeight: 6, top: 2, right: 10}),
+      yAxis: YAxis({
+        theme,
+        splitNumber: 3,
+        axisLabel: {fontSize: 11},
```

Comment:
does `fontSize` need to be a number? otherwise we could use the theme property `fontSizeExtraSmall`

Reply:
Let me give it a try :)

Reply:
![image](https://user-images.githubusercontent.com/1421724/111007113-f4054b80-8342-11eb-9a71-ec630f216a8b.png)

nope